src/index.html

```diff
@@ -203,34 +206,37 @@ <h2>HOBBIES & INTERESTS</h2>
       </ul>
 
       <!-- Skills -->
-      <section>
-        <h2>SKILLS</h2>
-        <ul>
-          <li>
-            <div>Microsoft Excel</div>
-            <div>Advanced</div>
-          </li>
-          <li>
-            <div>Microsoft Word</div>
-            <div>Advanced</div>
-          </li>
-          <li>
-            <div>Adobe</div>
-            <div>Proficient</div>
-          </li>
-          <li>
-            <div>Spanish</div>
-            <div>Advanced</div>
-          </li>
-          <li>
-            <div>PowerPoint</div>
-            <div>Proficient</div>
-          </li>
-          <li>
-            <div>French</div>
-            <div>Native</div>
-          </li>
-        </ul>
+      <section id="skills">
+        <div class="content">
+          <h2>SKILLS</h2>
+          <hr />
```

Comment:
This is not wrong. I think it does make some sense to have a `hr` here.
But I prefer just  user `:after` on h2 so you don't need to create extra html tag.